orderer/consensus/etcdraft/blockpuller.go

```diff
@@ -160,51 +157,49 @@ func NewBlockFetcher(support consensus.ConsenterSupport,
 		CensorshipSuspicionThreshold: time.Duration((int64(clusterConfig.ReplicationPullTimeout) * shuffleTimeoutPercentage / 100)),
```

Comment:
Can we export this to a configuration parameter in `orderer.yaml` ?

Reply:
can be done later.